core/templates/dev/head/css/oppia.css

```diff
@@ -2746,10 +2746,6 @@ div#ng-curtain {
   margin-bottom: 10px;
 }
 
-.oppia-notifications-tray {
```

Comment:
Is this CSS class being used anywhere else? If not, delete it from the html file too?